src/Symfony/Component/Cache/TaggableCacheInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache;
+
+/**
+ * Gets and stores items from a tag-aware cache.
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+interface TaggableCacheInterface extends CacheInterface
+{
+    /**
+     * @param callable(string, CacheItem):mixed $callback Should return the computed value for the given key/item
+     *
+     * @return mixed The value corresponding to the provided key
+     */
+    public function get(string $key, callable $callback);
```

Comment:
what's the point of defining those methods when they are the same as the inherited ones?

Reply:
They are not the same actually: the second argument to the closure is a CacheItemInterface in one case, and a CacheItem for taggable caches, in order to ship the tag() method when needed.

Reply:
Please add that to the phpdoc description. The difference is not obvious.

Reply:
comments updated